docs/diagnostic.md

```diff
@@ -0,0 +1,86 @@
+# Diagnostics
+
+One way to class AI models in Earth-2 MIP is by those that perform time-integration and those that do not.
```

Comment:
I think "The term diagnostic model refers here to models that do not make a timestep prediction, but simply predict some new channel(s) from existing ones at the same time"

Reply:
Updated! Thanks!